model/src/main/proto/spine_examples/shareaware/watchlist/events.proto

```diff
@@ -50,3 +51,16 @@ message WatchlistCreated {
     // The name of the created watchlist.
     string name = 3 [(required) = true];
 }
+
+// A share is watched from now.
+message ShareWatched {
+
+    // The ID of the watchlist where share is watched.
+    WatchlistId watchlist = 1;
+
+    // The ID of the user who was wanted the share to be watched
+    spine.core.UserId user = 2 [(required) = true];
+
+    // The share that is watched from now.
+    Share share = 3 [(required) = true];
```

Comment:
Is this a fully-fledged share? With price and everything?

If so, I don't think it should work like this. Instead, we should pass some identifier here, not all the bells-and-whistles.